src/lib/utils/loadstor.h

```diff
@@ -14,20 +14,6 @@
 #include <botan/internal/bswap.h>
 #include <vector>
 
-#if defined(BOTAN_TARGET_CPU_IS_BIG_ENDIAN)
```

Comment:
Not sure what this commit is doing here?

Reply:
Me neither. That's still open (the commit is also part of #3675). @falko-strenzke could you double-check this? I'm guessing that was some rebase mishap?

Reply:
yes, I must have included one commit too many during rebasing. removed it now.